backend/src/environment.ts

```diff
@@ -87,6 +98,16 @@ export class Environment {
     this.manufacturerDetails.product.appCode = MANUFACTURE_ID_STANDARD
   }
 
+  checkOob() {
+    lan.checkForWifiNetwork(REMOTEIT_PI_WIFI, Environment.updateOob)
+  }
+
+  //passsed to and called by wifi checking function
+  static updateOob(isActive: boolean) {
+    environment.oobActive = isActive
+    EventBus.emit(environment.EVENTS.send, environment.frontend)
+  }
```

Comment:
No need to be static. Please move to LAN class. Also the emit event is usually in the controller since that's what's triggering it.

Reply:
yeah, was because I did not know how to make it a promise.  Updated with the promise and moved to controller =)

Reply:
👍